src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/existing_class_def_2.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definition class="Symfony\Bundle\FrameworkBundle\Tests\Console\Descriptor\ClassWithoutDocComment" public="false" synthetic="false" lazy="false" shared="true" abstract="false" autowired="false" autoconfigured="false" file=""/>
```

Comment:
eol :P

Reply:
Ok, I think it's finally good, two blank lines for .txt fixtures, one blank line for other formats :)

Reply:
Oups, not all modifications pushed

Reply:
Sorry, my text editor made my mad ! It should be ok now